src/store/store.js

```diff
@@ -43,13 +43,14 @@ const migrations = {
       },
     };
   },
-  15: (/** @type {PersistedDefaultState} */ state) => {
+  17: (/** @type {PersistedDefaultState} */ state) => {
     return {
       ...state,
       settings: {
         ...state.settings,
         displayColumns: {
           ...state.settings.displayColumns,
+          futuresDailyBalance: [...initialState.settings.displayColumns.futuresDailyBalance],
           providerContracts: [...initialState.settings.displayColumns.providerContracts],
           providerOrders: [...initialState.settings.displayColumns.providerOrders],
```

Comment:
FYI I don't think you have to destructure the array here as well

Reply:
I think we should, otherwise it will just assign that reference, no?

Reply:
Done